de.metas.adempiere.adempiere/base/src/main/java-gen/org/compiere/model/X_C_BPartner_CreditLimit.java

```diff
@@ -0,0 +1,207 @@
+	}
+
+	/** Get Betrag.
+		@return Betrag in einer definierten Währung
+	  */
+	@Override
+	public java.math.BigDecimal getAmount () 
+	{
+		BigDecimal bd = (BigDecimal)get_Value(COLUMNNAME_Amount);
+		if (bd == null)
+			 return BigDecimal.ZERO;
+		return bd;
+	}
+
+	@Override
+	public org.compiere.model.I_AD_User getApprovedBy() throws RuntimeException
+	{
+		return get_ValueAsPO(COLUMNNAME_ApprovedBy_ID, org.compiere.model.I_AD_User.class);
+	}
+
+	@Override
+	public void setApprovedBy(org.compiere.model.I_AD_User ApprovedBy)
+	{
+		set_ValueFromPO(COLUMNNAME_ApprovedBy_ID, org.compiere.model.I_AD_User.class, ApprovedBy);
+	}
+
+	/** Set Approved By.
+		@param ApprovedBy_ID Approved By	  */
+	@Override
+	public void setApprovedBy_ID (int ApprovedBy_ID)
+	{
+		if (ApprovedBy_ID < 1) 
+			set_Value (COLUMNNAME_ApprovedBy_ID, null);
+		else 
+			set_Value (COLUMNNAME_ApprovedBy_ID, Integer.valueOf(ApprovedBy_ID));
+	}
+
+	/** Get Approved By.
+		@return Approved By	  */
+	@Override
+	public int getApprovedBy_ID () 
+	{
+		Integer ii = (Integer)get_Value(COLUMNNAME_ApprovedBy_ID);
+		if (ii == null)
+			 return 0;
+		return ii.intValue();
+	}
+
+	/** Set Bussines Partner Credit Limit.
+		@param C_BPartner_CreditLimit_ID Bussines Partner Credit Limit	  */
+	@Override
+	public void setC_BPartner_CreditLimit_ID (int C_BPartner_CreditLimit_ID)
+	{
+		if (C_BPartner_CreditLimit_ID < 1) 
+			set_ValueNoCheck (COLUMNNAME_C_BPartner_CreditLimit_ID, null);
+		else 
+			set_ValueNoCheck (COLUMNNAME_C_BPartner_CreditLimit_ID, Integer.valueOf(C_BPartner_CreditLimit_ID));
+	}
+
+	/** Get Bussines Partner Credit Limit.
+		@return Bussines Partner Credit Limit	  */
+	@Override
+	public int getC_BPartner_CreditLimit_ID () 
+	{
+		Integer ii = (Integer)get_Value(COLUMNNAME_C_BPartner_CreditLimit_ID);
+		if (ii == null)
+			 return 0;
+		return ii.intValue();
+	}
+
+	@Override
+	public org.compiere.model.I_C_BPartner getC_BPartner() throws RuntimeException
+	{
+		return get_ValueAsPO(COLUMNNAME_C_BPartner_ID, org.compiere.model.I_C_BPartner.class);
+	}
+
+	@Override
+	public void setC_BPartner(org.compiere.model.I_C_BPartner C_BPartner)
+	{
+		set_ValueFromPO(COLUMNNAME_C_BPartner_ID, org.compiere.model.I_C_BPartner.class, C_BPartner);
+	}
+
+	/** Set Geschäftspartner.
+		@param C_BPartner_ID 
+		Bezeichnet einen Geschäftspartner
+	  */
+	@Override
+	public void setC_BPartner_ID (int C_BPartner_ID)
+	{
+		if (C_BPartner_ID < 1) 
+			set_ValueNoCheck (COLUMNNAME_C_BPartner_ID, null);
+		else 
+			set_ValueNoCheck (COLUMNNAME_C_BPartner_ID, Integer.valueOf(C_BPartner_ID));
+	}
+
+	/** Get Geschäftspartner.
+		@return Bezeichnet einen Geschäftspartner
+	  */
+	@Override
+	public int getC_BPartner_ID () 
+	{
+		Integer ii = (Integer)get_Value(COLUMNNAME_C_BPartner_ID);
+		if (ii == null)
+			 return 0;
+		return ii.intValue();
+	}
+
+	/** Set Datum.
+		@param DateGeneral Datum	  */
+	@Override
+	public void setDateGeneral (java.sql.Timestamp DateGeneral)
```

Comment:
from the column name, i don't get what DateGeneral is about..
if i think it should rather be `validFrom`or `validTo`..if i correctly guessed that the date is about

Reply:
Checking with Mark